### PR TITLE
feat: docs for 0.27, general docs updates

### LINKED ISSUE #183: Secret for `local_uri` needs to be added to prod Secrets.toml for a successful deployment
When I use `shuttle_shared_db` I can make use of the `local_uri` parameter.
In conjunction with `shuttle-secrets` I can set this to `local_uri = "{secrets.DATABASE_URL}"`.

Even though the **local** uri is **only** used for local development, I need to set DATABASE_URL not only in `Secrets.dev.toml` but also in `Secrets.toml` 🤨

If the parameter is missing (at least defined with an empty string) it results in an error during the loading state:
> error="Failed to interpolate string. Is your Secrets.toml correct?"

Please add some clarification to the docs.

### PR DESCRIPTION
**Pull request by jonaro00**
Closes #183

### CODE REVIEW

File: examples/axum-static-files.mdx
Comment (oddgrd) [inline examples/axum-static-files.mdx:23]: :heart_eyes:

**Review by oddgrd — APPROVED** [review]
Looks great, thanks! We should remember to not merge this until after the release.

**Comment by jonaro00** [comment]
Can merge to develop now

**Pull request merged**
